### PR TITLE
scylla_ntp_setup: don't install ntpd package when it's already exists

### PR DESCRIPTION
**Pull request by syuu1228**
Don't install ntpd package when it's already exists.
Related with #7153

### CODE REVIEW

File: dist/common/scripts/scylla_ntp_setup [hunk 1]
```diff
@@ -41,7 +41,8 @@ if __name__ == '__main__':
     args = parser.parse_args()
 
     if is_debian_variant():
-        run('apt-get install -y ntp ntpdate')
+        if not os.path.exists('/usr/sbin/ntpd') or not os.path.exists('/usr/sbni/ntpdate'):
```
Comment (avikivity) [inline dist/common/scripts/scylla_ntp_setup:44]: s/sbin/sbin/

Comment (amoskong) [reply dist/common/scripts/scylla_ntp_setup:44]: s/sbni/sbin/

Comment (syuu1228) [reply dist/common/scripts/scylla_ntp_setup]: Fixed this.

**Comment by penberg** [comment]
Queued as 4deb245198f303938d8e69ba1189ae31ce93d9ab

**Pull request closed without merging**